elements/map/src/methods/map/setters.js

```diff
@@ -123,7 +123,7 @@ export function setControlsMethod(controls, oldControls, EOxMap) {
  * @return {Array<EoxLayer>}
```

Comment:
we can get rid of the "reverse" in the jsdoc function description

(same here: https://github.com/EOX-A/EOxElements/pull/1977/files#diff-88773987c26c0d60bb9ed9043cdf28fbe5c17b5f6ab414360f67efc343ba6c4eL138)

Reply:
done in https://github.com/EOX-A/EOxElements/pull/1977/commits/12595a4ee292ecf2b97aae867ef399e10a21c962